cogs/bot_online.py

```diff
@@ -0,0 +1,15 @@
+import discord
+from discord.ext import commands
+
+class Online(commands.Cog):

```

Comment:
This should be in the main bot file and doesn't need a class

Reply:
ok I will do that